src/app/components/Figure/ImageWrapper/index.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const ImageWrapper = styled.div`
```

Comment:
This probably doesn't need to be it's own component; can just add it inline to the Figure when the copyright is integrated.

Reply:
Talked this over with @sareh & @pjlee11, we've removed it from this PR & will add it to the Figure container when needed.